lib/galaxy/webapps/galaxy/api/users.py

```diff
@@ -74,11 +75,18 @@
 
 UserIdPathParam: DecodedDatabaseIdField = Path(..., title="User ID", description="The ID of the user to get.")
 APIKeyPathParam: str = Path(..., title="API Key", description="The API key of the user.")
+FlexibleUserIdPathParam: str = Path(..., title="User ID", description="The ID of the user to get or __current__.")
```

Comment:
I think the type should be a union of EncodedDatabaseId and the literal current here.